### PR TITLE
Fix missing RunTests.sh in CoreFX Linux jobs.

### LINKED ISSUE #23122: [arm64] error: No artifacts found
**Job:**
https://ci.dot.net/job/dotnet_coreclr/job/master/view/arm64/job/jitstress/job/arm64_cross_checked_ubuntu16.04_corefx_baseline_tst/129/

**Log:**
```
Archiving artifacts
ERROR: No artifacts found that match the file pattern "_/fx/artifacts/bin/**/testResults.xml". Configuration error?
ERROR: java.lang.InterruptedException: no matches found within 10000
Build step 'Flexible publish' changed build result to FAILURE
```

### PR DESCRIPTION
**Pull request by sandreenko**
We have to pass `/p:ArchiveTests=Tests` after recent scripts updates to generate `RunTests.sh` files.

Fixes #23122.

### CODE REVIEW

**Comment by sandreenko** [comment]
@dotnet-bot test Ubuntu16.04 arm64 Cross Checked corefx_baseline Build and Test

**Comment by sandreenko** [comment]
@dotnet-bot test Ubuntu16.04 arm64 Cross Checked corefx_baseline Build and Test

**Comment by sandreenko** [comment]
PTAL @BruceForstall, @jashook, @dotnet/jit-contrib.

Thanks @ViktorHofer for help.

File: tests/scripts/run-corefx-tests.py [hunk 1]
```diff
@@ -288,23 +288,26 @@ def main(args):
...
     if not Is_windows and arch == 'arm' :
         # We need to force clang5.0; we are building in a docker container that doesn't have
         # clang3.9, which is currently the default used by the native build.
-        build_args += ' /p:BuildNativeClang=--clang5.0'
+        common_config_args += ' /p:BuildNativeClang=--clang5.0'
 
     if not Is_windows and (arch == 'arm' or arch == 'arm64'):
         # It is needed under docker where LC_ALL is not configured.
-        build_args += ' --warnAsError false'
+        common_config_args += ' --warnAsError false'
```
Comment (ViktorHofer) [inline tests/scripts/run-corefx-tests.py]: If this is used for corefx then use a single hyphen instead of double.

**Pull request merged**